src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1336,7 +1336,7 @@ public static void fixNullStructureReferences(ChunkAccess chunk, Map<StructureFe
             if (!currentDimNames.containsAll(VANILLA_DIMS))
             {
                 LOGGER.warn("Detected missing vanilla dimensions from the world!");
-                RegistryAccess regs = ObfuscationReflectionHelper.getPrivateValue(RegistryReadOps.class, ops, "f_13563" + "6_");
+                RegistryAccess regs = ObfuscationReflectionHelper.getPrivateValue(RegistryReadOps.class, ops, "f_179860_");
```

Comment:
I would recommend you follow the established convention used throughout Forge for `ObfuscationReflectionHelper` which would mean `"f_179860_"` becomes `"f_17986" + "0_"`.